packages/webcert/src/feature/subscription/SubscriptionWarningModal.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { CustomButton, ModalBase, ResourceLinkType } from '@frontend/common'
+import { getUserResourceLinks } from '../../store/user/userSelectors'
+import { acknowledgeSubscription } from '../../store/user/userActions'
+import WCDynamicLink from '../../utils/WCDynamicLink'
+
+const ModalContent = () => {
+  return (
+    <div>
+      <p className="iu-mb-1rem">Du ser detta meddelande för att din verksamhet saknar abonnemang för Webcert.</p>
+
+      <p className="iu-mb-1rem">
+        För att er verksamhet fortsatt ska kunna utfärda intyg efter <span className="iu-color-error">2022-05-10</span> kräver Inera att ni
+        tecknar kundavtal samt beställer abonnemang för tjänsten Webcert.
+      </p>
+
+      <p className="iu-mb-1rem">
+        Om abonnemang saknas efter <span className="iu-color-error">2022-05-10</span> kommer inga nya intyg att kunna utfärdas. Er
+        verksamhet får då endast tillgång till Webcert för att läsa tidigare utfärdade intyg.
```

Comment:
I think this text reads a little off. Are we sure this is how we want to phrase the information?

Reply:
I think this text will change when the requirements are set.